<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户端</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab0">
    <attribute name="title">
     <string>欢迎使用</string>
    </attribute>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>欢迎使用FedoraMeow点菜系统～～～～</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>240</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>祝您使用愉快～～～～～～～～～</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>359</width>
       <height>104</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>使用指南：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>1.请现在聊天界面连接到服务器端以后再进行别的操作</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>2.点菜前请先在“查询和点菜“界面中申请桌号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>3.服务界面可以实现结帐、呼叫服务员、催菜功能</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>聊天界面</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>271</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>280</y>
       <width>403</width>
       <height>107</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>对方IP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lEIP">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>端口号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lEPort">
        <property name="text">
         <string>3333</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pBConn">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pBAbort">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>断开连接</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>编辑栏</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="lEMsg"/>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pBSend">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>发送消息</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>查询和点菜</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>331</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>查询和点菜窗口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ordButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>320</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>一键下单！</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>429</x>
       <y>10</y>
       <width>261</width>
       <height>140</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="quecookButton">
        <property name="text">
         <string>查询所有厨师信息</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="quedishButton">
        <property name="text">
         <string>查询所有菜品信息</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="squecookEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="squecookButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>输入厨师名字查询</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="squedishEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="squedishButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>输入菜品名字查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>160</y>
       <width>261</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>人数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="peonumEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="tabidButton">
        <property name="text">
         <string>申请桌号</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="tableEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入桌号：</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>310</y>
       <width>261</width>
       <height>97</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>请输入菜品编号和桌号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="del1Edit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="del2Edit"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="delButton">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>200</y>
       <width>270</width>
       <height>97</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>请输入桌号和菜品编号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="add1Edit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="add2Edit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>加入</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab3">
    <attribute name="title">
     <string>服务界面</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser3">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>21</y>
       <width>341</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>323</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>结算</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>桌号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="monEdit1"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>付钱：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="monEdit2"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="monButton">
        <property name="text">
         <string>买单</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>如果您没带够钱，请输入为您买单的桌号：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="monEdit3"/>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="monButton2">
        <property name="text">
         <string>代付</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>180</y>
       <width>323</width>
       <height>97</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>催菜/退菜</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>桌号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="calltabEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>菜号：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="calldishEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="decButton">
        <property name="text">
         <string>退菜</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="callButton">
        <property name="text">
         <string>催菜</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>290</y>
       <width>323</width>
       <height>97</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>告知上菜情况</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>桌号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="gettabEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>菜号：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="getdishEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="getButton">
        <property name="text">
         <string>已上菜</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>390</y>
       <width>321</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>呼叫服务员：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="callsevEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="callsevButton">
        <property name="text">
         <string>呼叫</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lEMsg</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>EnableSendButton(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>EnableSendButton(QString)</slot>
 </slots>
</ui>
